lib/modules/manager/tekton/readme.md

```diff
@@ -25,6 +25,24 @@ There are three ways to use a Tekton Bundle reference:
 
 Renovate's Tekton manager supports all the methods listed above.
 
+### Configuring images in Tekton Tasks
+
+You can configure the container images that Tekton uses when it runs tasks.
+You may use these attributes to configure images:
+
+1. In a Task step
+1. In a Task stepTemplate
+1. In a Task sidecar
```

Comment:
Hmm, I don't like how we're repeating the word `In` in the list items. Can we rewrite to move it into the introductory sentence, or drop it? I'm looking for a structure like this:

> You may use ... to ... in:
> 
> - List item 1
> - List item 2
> - List item 3

Reply:
Yeah. That looks better. Updated. Let me know what you think!